api/mysagw/identity/export.py

```diff
@@ -31,37 +31,49 @@ def _fetch_related_strings(self, manager, field):
             m2m_strings.append(self._str_or_empty(getattr(entry, field)))
         return m2m_strings
 
-    def fetch_identity_data(self, identity):
+    def fetch_identity_data(self, identity, exclude_fields=None):
```

Comment:
The `exclude_fields` param seems to complicate things by quite a bit. Maybe (if not too computationally intensive) we could live with filtering the resulting dict just before returning instead?

Reply:
As discussed.